lgsm/functions/update_mumble.sh

```diff
@@ -15,7 +15,6 @@ fn_update_mumble_dl(){
 	echo -e "copying to ${filesdir}...\c"
 	fn_script_log "Copying to ${filesdir}"
 	cp -R "${lgsmdir}/tmp/murmur-static_${mumblearch}-${availablebuild}/"* "${filesdir}"
-	rm -R "${lgsmdir}/tmp"
```

Comment:
The tmp dir does not get removed currently. However that may change at some point